.eslintrc.js

```
@@ -5,6 +5,9 @@ module.exports = {
   ],
   parser: 'babel-eslint',
   rules: {
+    'class-methods-use-this': 0,
+    'import/no-named-as-default': 0,
```

`import/no-named-as-default` should be enabled, it's likely misleading.